packages/protocol/src/data/form.ts

```diff
@@ -16,13 +17,56 @@ export type TextIconStyle = "INLINE" | "BLOCK";
 export type TextType = "RAW" | "MARKDOWN";
 
 export interface Form {
+  cmsMetaRequest: CmsMetaRequest;
+  compositeInfo: CompositeInfo[];
+  contentObject: ContentObject[];
+  form: Form1;
```

Comment:
as the form is already in use I think we should name this `Form` to something like `FormSchema`